tools/test-staking.ts

```diff
@@ -185,7 +185,8 @@ async function test() {
 
   // Revoke Nomination
   const unsub3 = await polkadotApi.tx.parachainStaking
-    .leaveNominators()
+    .revokeNomination(GERALD) //TODO: when converting to test add .leaveNominators()
+    // that should produce the same behavior
```

Comment:
Has this been resolved?

Reply:
resolved, does produce the same behavior